sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## 1.2.0-beta.3 (Unreleased)
 
+### Breaking Changes
```

Comment:
Will this go out as a beta release i.e. 1.2.0-beta.3? If not, then these breaking changes don't apply to a GA release (1.3.0). The change list on a GA release should ideally not have anything in the "Breaking changes" section as it can tip off customers from upgrading.